common/scripts/upgrade_to_continuous_delivery.sh

```diff
@@ -180,6 +184,7 @@ function switch_to_continous_delivery() {
     done < <(oc get sub --all-namespaces | grep ibm-common-service-operator | awk '{print $1" "$2}')
     
     success "Updated all ibm-common-service-operator subscriptions successfully."
+    info "Please wait for a moment to let ibm-common-service-operator upgrade all foundational services."
```

Comment:
Just an advice. Does the following message look better?
```
Please wait a moment for ibm-common-service-operator to upgrade all foundational services
```

Reply:
It looks good! Updated.

Reply:
Thanks!